inc/config.class.php

```diff
@@ -3475,6 +3480,206 @@ static function showLogsInterval($name, $value, $options = []) {
       echo $out;
    }
 
+   /**
+    * Security policy form
+    *
+    * @since 9.5.0
+    *
+    * @return void
+    */
+   function showFormSecurity() {
+      global $CFG_GLPI;
+
+      if (!Config::canUpdate()) {
+         return false;
+      }
+
+      $rand = mt_rand();
+
+      echo '<div class="center" id="tabsbody">';
+      echo '<form name="form" action="' . Toolbox::getItemTypeFormURL(__CLASS__) . '" method="post">';
+      echo '<table class="tab_cadre_fixe">';
+      echo '<tr><th colspan="4">' . __('Security setup') . '</th></tr>';
+
+      echo '<tr class="tab_bg_1">';
+      echo '<td colspan="4" class="center b">' . __('Password security policy') . '</td>';
+      echo '</tr>';
+
+      echo '<tr class="tab_bg_2">';
```

Comment:
This field (among others) is present in two different forms; should not it be removed from its original location?

Reply:
Yes, it had to be removed. It is done now.